tests/test_versions.py

```diff
@@ -59,6 +59,22 @@ def setUp(self):
         except Exception:
             raise unittest.SkipTest("Git repository for this package is not accessible.")
 
+    def test_get_lsst_version(self):
+        # test get_lsst_version which returns version for the current directory
+        datadir = os.path.join(TESTDIR, "data")
+        # test for directory with package info
+        os.chdir(os.path.join(datadir, "something.egg-info"))
+        version = get_lsst_version()
+        self.assertEqual(version, "1.1.0")
+        # test for git repo
+        os.chdir(GITDIR)
```

Comment:
chdir in tests is not a good idea because if the call below fails it won't chdir back. Better to use the optional dir parameter that will be added.

Reply:
Changed